app/src/androidTest/java/org/simple/clinic/patient/businessid/BusinessIdMetaDataAdapterAndroidTest.kt

```diff
@@ -41,6 +41,10 @@ class BusinessIdMetaDataAdapterAndroidTest {
                 assigningUserUuid = UUID.randomUUID(),
                 assigningFacilityUuid = UUID.randomUUID()
             )
+            BusinessId.MetaDataVersion.MedicalRecordNumberMetaDataV1 -> BusinessIdMetaData.MedicalRecordNumberMetaDataV1(
+                assigningUserUuid = UUID.randomUUID(),
+                assigningFacilityUuid = UUID.randomUUID()
```

Comment:
Can we change this (and the rest of the class) to use fixed UUID instances instead of random ones, please? This is part of an ongoing effort to reduce the amount of non-determinism in our tests.

I also do not know if you are aware of this, but we have a tooling plugin which adds live templates to make this easier: https://github.com/simpledotorg/simple-android/tree/master/tooling/intellij-plugin

Reply:
Can this be done by using `FakeUuidGenerator` class ?

Reply:
That's not meant for this use case. That class is meant for use cases where we want to use a fixed UUID wherever we need to generate a random UUID in production code.

I'd suggest installing the plugin I linked since that has a live template which generates random UUIDs to use in tests.